frontend/src/i18n/en-US.json

```
@@ -51,7 +51,14 @@
   "admin.licenseInfo.getLicense": "Get License",
   "admin.licenseInfo.selfHostedNoLicense.description": "Thank you for using Cryptomator Hub! You have been granted the Community License. If you need more seats, upgrade your license.",
   "admin.licenseInfo.managedNoLicense.description": "Thank you for using Cryptomator Hub! You currently have no active license.",
-
+  "admin.webOfTrust.title": "Web of Trust",
+  "admin.webOfTrust.description": "Configure the Web of Trust settings to manage how trust is established between users in the system. These settings affect the verification of user identities and key signatures.",
+  "admin.webOfTrust.trustLevel.error": "Trust level must be between 0 and 10.",
+  "admin.webOfTrust.trustLevel.title": "Trust Level",
+  "admin.webOfTrust.maxChainLength.error": "Maximum chain length must be greater than 0.",
+  "admin.webOfTrust.maxChainLength.title": "Maximum Chain Length",
```

Please adjust the names. Furthermore, we probably need to explain the meaning of both values, so you should add some description field.

```suggestion
  "admin.webOfTrust.wotMaxDepth.error": "Maximum WoT Depth must be between 0 and 10.",
  "admin.webOfTrust.wotMaxDepth.title": "Maximum WoT Depth",
  "admin.webOfTrust.wotMaxDepth.description": "Maximum distance between two users, so an indirect acquaintanceship is still considered trustworthy (assuming intermediate identities have been verified).",
  "admin.webOfTrust.wotIdVerifyLen.error": "Fingerprint Verification Preciseness must be positive.",
  "admin.webOfTrust.wotIdVerifyLen.title": "Fingerprint Verification Preciseness",
  "admin.webOfTrust.wotIdVerifyLen.description": "How many digits of another users fingerprint one needs to enter to verify their identity.",
```